x/committee/types/permissions.go

```diff
@@ -405,6 +427,7 @@ func NewAllowedCollateralParam(
 	}
 }
 
+// Allows implement permission interface
```

Comment:
nit: these methods don't implement the `types.Permission` interface (different arg types). Only the top level permission methods like `func (perm SubParamChangePermission) Allows(...` implements it